ansible/vars/icds/icds_public.yml

```diff
@@ -351,13 +351,6 @@ localsettings:
             - 'static-ccs_record_cases_monthly'
             - 'static-ccs_record_cases_monthly_v2'
             - 'static-ccs_record_cases_monthly_tableau_v2'
-      - name: 'kafka-ucr-static-person_cases'
-        class: 'corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow'
-        instance: 'corehq.apps.userreports.pillow.get_kafka_ucr_static_pillow'
-        params:
-          topics:
-            - 'case-sql'
-          include_ucrs:
             - 'static-person_cases_v2'
```

Comment:
No. that should be included in the other pillow. I guess it would have been a better title to "combine person pillow with case pillow"